src/libraries/System.Security.Cryptography.Encoding/src/Resources/Strings.resx

```diff
@@ -117,6 +117,15 @@
   <data name="Argument_InvalidValue" xml:space="preserve">
     <value>Value was invalid.</value>
   </data>
+  <data name="Argument_PemEncoding_NoPemFound" xml:space="preserve">
+    <value>No PEM encoded data found.</value>
```

Comment:
Should we explicitly state what PEM stands for in the exception message/API docs, or is it expected for the caller to have this domain knowledge?

Something like:
```suggestion
    <value>No Privacy-Enhanced Mail (PEM) encoded data found.</value>
```

Reply:
Nah, it'd be like if every JSON exception said "JavaScript Object Notation (JSON)".  The acronym explains the origin, but itself isn't relevant anymore.